server/models/user.model.js

```diff
@@ -45,8 +45,8 @@ import mongoose from 'mongoose';
  *           phone: 08012345678
  *           phone2: 08012345678
  *           address: {street1: 1 sesame street, street2: 12 solomon close, city: Ikeja, state: Lagos, country: Nigeria}
- *           preferences: {type: 3 bedroom apartment, location: lekki phase 1, maxPrice: 50000000, minPrice: 15000000}
- *           notification: ['Your account has been activated']
+ *           preferences: {houseType: 3 bedroom apartment, location: lekki phase 1, maxPrice: 50000000, minPrice: 15000000}
+ *           notification: [description: Your account has been activated, type: info, URL: app.ballers.ng/dashboard, status: 0, dateAdded: 22-03-2021]
```

Comment:
Date should be in YYYY-mm-dd format.
```suggestion
 *           notification: [description: Your account has been activated, type: info, URL: app.ballers.ng/dashboard, status: 0, dateAdded: 2021-03-22]
```